configure.ac

```diff
@@ -238,7 +238,12 @@ AS_IF([test "x$with_nvdimm" != "xno"],
       [])
 
 AS_IF([test "x$with_nvme" != "xno"],
-      [LIBBLOCKDEV_PKG_CHECK_MODULES([NVME], [libnvme >= 1.2])],
+      [LIBBLOCKDEV_PKG_CHECK_MODULES([NVME], [libnvme >= 1.2])
+      AS_IF([$PKG_CONFIG --atleast-version=1.3 libnvme],
+            [AC_DEFINE([HAVE_LIBNVME_1_3])], [])
```

Comment:
Debian has libnvme 1.3 in testing, we have 1.4 in Fedora 38 and CentOS 9 Stream so I think we might just require 1.3 and have one less check here.

Reply:
True, we still haven't released anything and it will take some more time :-)

Reply:
Fixed.